vcpkg/action.yml

```diff
@@ -7,42 +7,19 @@ inputs:
     description: "Path to vcpkg-configuration.json"
     required: false
     default: "./vcpkg-configuration.json"
-  vcpkg-root:
-    description: "Root folder for vcpkg"
-    default: "${{ github.workspace }}/.vcpkg"
-    required: false
-  cache:
-    description: "Cache key suffix, set to empty to disable cache."
-    default: "-"
-    required: false
   
 runs:
   using: 'composite'
   steps:
-    - name: Cache vcpkg
```

Comment:
Removing the cache means that all the artifacts are downloaded all over again.

Reply:
Is it logical to cache tool artifacts when the tool is readily available and pre-installed in the GitHub runner image?